komga-webui/src/locales/en.json

```diff
@@ -557,32 +557,32 @@
     }
   },
   "error_codes": {
-    "ERR_1000": "File could not be accessed during analysis",
-    "ERR_1001": "Media type is not supported",
+    "ERR_1000": "Could not access file during analysis",
+    "ERR_1001": "Unsupported media type",
     "ERR_1002": "Encrypted RAR archives are not supported",
     "ERR_1003": "Solid RAR archives are not supported",
     "ERR_1004": "Multi-Volume RAR archives are not supported",
     "ERR_1005": "Unknown error while analyzing book",
-    "ERR_1006": "Book does not contain any page",
-    "ERR_1007": "Some entries could not be analyzed",
-    "ERR_1008": "Unknown error while getting book's entries",
+    "ERR_1006": "There are no pages in the book",
+    "ERR_1007": "Could not analyze some pages",
+    "ERR_1008": "Could not fetch the book's entries, for unknown reason",
     "ERR_1009": "A read list with that name already exists",
-    "ERR_1010": "No books were matched within the read list request",
-    "ERR_1011": "No unique match for series",
-    "ERR_1012": "No match for series",
-    "ERR_1013": "No unique match for book number within series",
+    "ERR_1010": "Could not find any books from read list request",
+    "ERR_1011": "Could not find that specific book in the series",
+    "ERR_1012": "Could not find anything like that in the series",
+    "ERR_1013": "Could not find this book number in the series",
```

Comment:
You changed the meaning of ERR_1011.

Your ERR_1012 doesn't mean anything anymore.

You changed the meaning of ERR_1013. It says it cannot find a unique match for that book number, you are saying it cannot find that book number. That's different.